src/components/HoneySwapRoute/HoneySwapRoute.tsx

```diff
@@ -113,15 +113,15 @@ const HoneySwapRoute = ({
   const getFormattedAmountByTokenAndOffer = (
     token?: IAssetWithBalance | null,
     offer?: ExchangeOffer | null,
-    tokenAmount?: string | null
+    tokenAmount?: string | null,
+    state: string = 'one'
   ) => {
     if (token && token.address !== GNOSIS_USDC_CONTRACT_ADDRESS && !!offer) {
       return formatAmountDisplay(
         Number(ethers.utils.formatUnits(offer.receiveAmount.toString(), token.decimals)).toFixed(4)
       );
-    } else if (token && token.address === GNOSIS_USDC_CONTRACT_ADDRESS && !offer && !!tokenAmount) {
-      console.log('tokenAmount', tokenAmount);
-      return formatAmountDisplay(Number(tokenAmount).toFixed(4));
+    } else if (token && token.address === GNOSIS_USDC_CONTRACT_ADDRESS && !!tokenAmount) {
+      return (Number(tokenAmount) * 10 ** (state === 'one' ? 0 : 12)).toFixed(2);
```

Comment:
I think if we're using a numerical value for `state`, we should just be using integers instead of strings

Reply:
yeah